src/Components/SideBar.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import {
+  MdThumbUp,
+  MdHistory,
+  MdWatchLater,
+  MdHome,
+  MdPlaylistPlay
+} from "react-icons/md";
+import { Link } from "react-router-dom";
+export function SideBar() {
+  return (
+    <div style={{ width: "250px" }} className="position-absolute">
+      <div style={{ fontSize: "1.4rem" }} className="flex-column">
+        <div className="">
```

Comment:
Please take a look at empty classNames in this file.

Reply:
If you do not require it, then remove it.

Reply:
yes I have removed it.

Reply:
looks good.